scim-spec/scim-spec-schema/src/main/java/org/apache/directory/scim/spec/schema/Schemas.java

```diff
@@ -201,6 +202,7 @@ private static List<Schema.Attribute> createAttributes(String urn, List<Field> f
       log.debug("Attempting to set the attribute type, raw value = " + typeName);
       switch (typeName) {
         case STRING_TYPE_IDENTIFIER:
+        case STRING_TYPE:
```

Comment:
I need to add an issue to make coverage is good here.
These issues likely only affect Extensions, but that is still important.

The issue I was seeing is one of the Test object Extensions has a list of strings. Without this fix that list was, the list was showing up as a complex type, and would not diff correctly

Reply:
Not related to this PR, but there was a similar issue in #425
Added more robust fix and tests in #452